System.Web/Remotion.ThirdPartyDependency.System.Web.csproj

```
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\_._">
+      <Pack>true</Pack>
+      <PackagePath>\</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <Owner>RUBICON IT GmbH</Owner>
+    <Version>4.8.0</Version>
+    <Title>System.Web</Title>
+    <Description>System.Web of .NET Framework</Description>
```

```suggestion
    <Description>System.Web of .NET Framework</Description>
```
Not sure about the description. Perhaps list of Assembly names?
Systme.Web
System.Web.Extensions
System.Web.Services
a couple more

---

I don't know how much of coreforms is based on System Web, I put in some things that I could find in the reference sources on github and in coreforms, not sure about any other parts.

---

if you start the list (see above), I can complete it.

---

Temp commit for list is online :)

---

Sent a comment.